3-Authorization-II/3-App-CallAPI/daemon/src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+spring:
+    main:
+        web-application-type: none
+    security:
+        oauth2:
+            client:
+                provider:
+                    azure:
+                        token-uri: https://login.microsoftonline.com/Enter_Your_Tenant_ID_Here/oauth2/v2.0/token
+                registration:
+                    azure:
+                        authorization-grant-type: client_credentials
+                        client-authentication-method: CLIENT_SECRET_POST
+                        client-id: Enter_Your_Client_ID_Here
+                        client-secret: Enter_Your_Client_Secret_Here
+                        scope: api://Enter_Your_WebAPI_Client_ID_Here/.default
+app:
+  api:
+    base-address: http://localhost:8082
```

Comment:
If possible, we should list the API endpoints we call in this file, then use those in the source code